### ISSUE TITLE
[Bug]: Sp-slider lacks 3:1 contrast ratio between foreground and background

### ISSUE THREAD

**Issue by Himanshu-998** [adobe/spectrum-web-components#4582]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

sp-slider

### Expected behavior

The contrast ratio should be atleast 3:1 between foreground and background of slider

### Actual behavior

The contrast ratio is not be atleast 3:1 between foreground and background of slider

### Screenshots

<img width="1566" alt="Screenshot 2024-07-02 at 10 38 35 AM" src="https://github.com/adobe/spectrum-web-components/assets/39983976/d0840baa-007d-4630-a6ca-cea9ace1b5eb">

**Comment by najikahalsema** [comment]
We won't fix this as we are not making any more visual update to Spectrum 1. When we release S2, this will be fixed.
